src/vivarium_nih_us_cvd/components/observers.py

```diff
@@ -21,11 +24,24 @@ class ResultsStratifier(ResultsStratifier_):
 
     def setup(self, builder: Builder) -> None:
         super().setup(builder)
-        age_bins = self.age_bins
```

Comment:
This is the same logic I've been using to adjust age bins but moves it more appropriately directly into the _get_age_bins method